js/constants/appConfig.js

```diff
@@ -6,7 +6,7 @@ const { getTargetAboutUrl } = require('../lib/appUrlUtil')
 // BRAVE_UPDATE_HOST should be set to the host name for the auto-updater server
 const updateHost = process.env.BRAVE_UPDATE_HOST || 'https://brave-laptop-updates.global.ssl.fastly.net'
 const winUpdateHost = process.env.BRAVE_WIN_UPDATE_HOST || 'https://brave-download.global.ssl.fastly.net'
-const crashURL = process.env.BRAVE_CRASH_URL || 'https://laptop-updates.brave.com/1/crashes'
+const crashURL = process.env.BRAVE_CRASH_URL || 'https://brave-laptop-updates.herokuapp.com/1/crashes'
```

Comment:
I don't think it needs to be added to the CSP as the HTTP request is issued by breakpad (separate process). But it probably wouldn't hurt to add it.